main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ParentStationValidator.java

```diff
@@ -156,4 +186,29 @@ static class WrongParentLocationTypeNotice extends ValidationNotice {
       this.expectedLocationType = expectedLocationType;
     }
   }
+
+  /**
+   * Unused parent station.
+   *
+   * <p>A stop has `location_type` STATION (1) but does not appear in any stop's `parent_station`.
+   */
+  @GtfsValidationNotice(
+          severity = SeverityLevel.INFO,
+          files = @FileRefs({GtfsStopSchema.class}))
```

Comment:
@bradyhunsaker you can try this:
```suggestion
  @GtfsValidationNotice(severity = SeverityLevel.INFO, files = @FileRefs({GtfsStopSchema.class}))
```
* also make sure SeverityLevel is imported

Reply:
I changed it to import INFO, as it already does with ERROR.